### PR TITLE
Fix balance coin icon is off center

### PR DESCRIPTION
**Pull request by osdnk**
Closes https://linear.app/rainbow/issue/RNBW-1544/balance-coin-icon-is-off-center

### CODE REVIEW

**Comment by osdnk** [comment]
Looks like ellipsis on android makes some weirdness w/o setting up the height

File: src/components/token-info/TokenInfoBalanceValue.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import styled from 'styled-components';
 import { CoinIcon } from '../coin-icon';
 import { RowWithMargins } from '../layout';
 import TokenInfoValue from './TokenInfoValue';
 import { useColorForAsset } from '@rainbow-me/hooks';
 import { magicMemo } from '@rainbow-me/utils';
 
+const InfoValue = styled(TokenInfoValue)`
+  ${android ? 'height: 37.7;' : ''}
```
Comment (skylarbarrera) [inline src/components/token-info/TokenInfoBalanceValue.js:10]: android &&

**Review by skylarbarrera — COMMENTED** [review]
💪

**Review by skylarbarrera — APPROVED** [review]
lgtm rerunning CI, nice catch

**Pull request merged**